packages/app/src/app/overmind/effects/api/apiFactory.ts

```diff
@@ -140,13 +140,22 @@ function getMessage(error: AxiosError) {
         error.message = errors; // eslint-disable-line no-param-reassign
       }
     } else if (response.data.error) {
-      error.message = response.data.error; // eslint-disable-line no-param-reassign
+      const { error_code, message, ...data } = response.data.error as {
+        message: string;
+        error_code: string;
+        [k: string]: any;
+      };
+      // @ts-ignore
+      error.error_code = error_code; // eslint-disable-line no-param-reassign
+      error.message = message; // eslint-disable-line no-param-reassign
+      // @ts-ignore
+      error.data = data; // eslint-disable-line no-param-reassign
```

Comment:
A lot of overrides here, instead of "convert" could you just say "createError" and create a new one based on the AxiosError? Cause you expect a returned result which you continue using